static/app/views/organizationStats/index.tsx

```diff
@@ -149,7 +149,10 @@ export class OrganizationStats extends Component<Props> {
 
   // Project selection from GlobalSelectionHeader
   get projectIds(): number[] {
-    return this.hasProjectStats ? this.props.selection.projects : [];
+    const selection_projects = this.props.selection.projects.length
+      ? this.props.selection.projects
+      : [-1];
+    return this.hasProjectStats ? selection_projects : [-1];
```

Comment:
We should generally try to avoid magic numbers. We represent this with the constant `ALL_ACCESS_PROJECTS`

```suggestion
      : [ALL_ACCESS_PROJECTS];
    return this.hasProjectStats ? selection_projects : [ALL_ACCESS_PROJECTS];
```

And it can be imported like this `import {ALL_ACCESS_PROJECTS} from 'sentry/constants/pageFilters';`

Reply:
ah nice - thank you!